### DISCUSSION TITLE
Light source/emitter

### DISCUSSION THREAD

**Discussion by PyroTech03** [AscensionGameDev/Intersect-Engine#2035] [Ideas]
Would be great to have gear or spells be able to emit proper light like a torch or flashlight. Right now, there is no good reason to set darkness too high as it makes the game largely unplayable. And if you set your player light level radius too high, then you negate the purpose of the dark.

**Comment by da-rulez** [comment]
Is there not an equipment animation you can use for that?

**Reply by PyroTech03** [reply]
I haven't tried that as I just saw how that's used in the last day or so. Though that I thought was for animating the sprite of the equipment.

**Reply by da-rulez** [reply]
Yeah but pretty sure you can make an empty animation and just include the light to render on the paperdolled torch
